docs/appendix/jupyterlab_extensions.rst

```diff
@@ -0,0 +1,181 @@
+Porting nbextensions to JupyterLab
+==================================
+
+We have made good progress porting the extensions to JupyterLab, but we are not quite finished. This document contains notes on the progress for all of the extensions.
+
+Assignment List
+---------------
+
+The assignment list JupyterLab extension contains the exact same functionality and layout as the nbextension. After installation it can be launched by opening the command palette on the left side and searching for Assignment List
```

Comment:
```suggestion
The assignment list JupyterLab extension contains the exact same functionality and layout as the nbextension. After installation, it can be launched by opening the command palette on the left side and searching for Assignment List.
```